### PR TITLE
DOCSP-29458 Edit x509-authentication-database-user

### PR DESCRIPTION
**Pull request by pierwill**
## Proposed changes

_Jira ticket:_ https://jira.mongodb.org/browse/DOCSP-29458

Edit documentation for X509 certificates resources.

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Manual QA performed:

- [ ] cfn invoke for each of CRUDL/cfn test
- [ ] Updated resource in  [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples)
- [ ] Published to AWS private registry
- [ ] Used the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create and update a stack in AWS
- [ ] Deleted stack to ensure resources are deleted
- [ ] Created multiple resources in same stack
- [ ] Validated in Atlas UI
- [ ] Included screenshots

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] For CFN Resources: I have released by changes in the private registry and proved by change
  works in Atlas

### CODE REVIEW

File: cfn-resources/x509-authentication-database-user/README.md [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
 
-## Attributes & Parameters
+Returns, edits, and removes user-managed X.509 configurations.
+Also returns and generates MongoDB Atlas-managed X.509 certificates for database users.
 
-Please consult the [Resource Docs](docs/README.md)
+## Attributes and Parameters
 
-## Local Testing
+See the [Resource Docs](docs/README.md).
```
Comment (andreaangiolillo) [inline cfn-resources/x509-authentication-database-user/README.md:9]: this link `docs/README.md` is not correct

File: cfn-resources/x509-authentication-database-user/README.md [hunk 2]
```diff
@@ -1,61 +1,74 @@
...
-echo "Sample request:"
-cat test.request.json
-cfn invoke CREATE test.request.json 
-cfn invoke DELETE test.request.json 
-```
-Both CREATE & DELETE tests must pass.
 ## Cloudformation Examples
 
-Please see the [CFN Template](/examples/x509-authentication-db-user/x509-authentication-db-user.json) for example resource
+See the [CFN Template](/examples/x509-authentication-db-user/x509-authentication-db-user.json) for example resource.
```
Comment (andreaangiolillo) [inline cfn-resources/x509-authentication-database-user/README.md:13]: this link is not correct

File: cfn-resources/x509-authentication-database-user/README.md [hunk 3]
```diff
@@ -1,61 +1,74 @@
...
+cfn invoke DELETE test.request.json
+```
+
+The `CREATE` and `DELETE` tests must pass.
+
 ## For More Information
-See the MongoDB Atlas API [X.509 Authentication for Database Users Endpoint](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/X.509-Authentication-for-Database-Users)
\ No newline at end of file
+
+See the MongoDB Atlas API [X.509 Authentication for Database Users Endpoint](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/X.509-Authentication-for-Database-Users)
```
Comment (andreaangiolillo) [inline cfn-resources/x509-authentication-database-user/README.md]: you can remove this part and add a link to https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/x509-authentication-database-user/test/README.md

File: cfn-resources/x509-authentication-database-user/README.md [hunk 4]
```diff
@@ -1,61 +1,74 @@
...
 
-The [/quickstart-mongodb-atlas/scripts/launch-x-quickstart.sh](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/quickstart-mongodb-atlas/scripts/launch-x-quickstart.sh) script
-can be used to safely inject your MongoDB Cloud ApiKey environment variables into an example
+The [`launch-x-quickstart.sh`](https://github.com/aws-quickstart/quickstart-mongodb-atlas/blob/main/scripts/launch-x-quickstart.sh) script
+can be used to safely inject your MongoDB Atlas ApiKey environment variables into an example
 CloudFormation stack template along with the other necessary parameters.
 
-You can use the x509authenticationdatabaseuser.sample-template.yaml to create a stack using the resource.
-Similar to the local testing described above you can follow the logs for the deployed
+You can use the`x509authenticationdatabaseuser.sample-template.yaml` to create a stack using the resource.
```
Comment (andreaangiolillo) [inline cfn-resources/x509-authentication-database-user/README.md:30]:
```suggestion
You can use the [x509authenticationdatabaseuser.sample-template.yaml](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/examples/x509-authentication-db-user/x509-authentication-db-user.json) to create a stack using the resource.
```

File: cfn-resources/x509-authentication-database-user/README.md [hunk 5]
```diff
@@ -1,61 +1,74 @@
...
+```
 TAGS=logging make
 cfn submit --verbose --set-default
+```
 
 ## Usage
 
-The [/quickstart-mongodb-atlas/scripts/launch-x-quickstart.sh](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/quickstart-mongodb-atlas/scripts/launch-x-quickstart.sh) script
-can be used to safely inject your MongoDB Cloud ApiKey environment variables into an example
+The [`launch-x-quickstart.sh`](https://github.com/aws-quickstart/quickstart-mongodb-atlas/blob/main/scripts/launch-x-quickstart.sh) script
```
[the remark this answers is not in the document]
Comment (pierwill) [reply cfn-resources/x509-authentication-database-user/README.md:26]: Where should this link point to?

File: cfn-resources/x509-authentication-database-user/README.md (hunk 2 above)
Comment (pierwill) [reply cfn-resources/x509-authentication-database-user/README.md:13]: Where should this link point to?

**Review by andreaangiolillo — APPROVED** [review]

**Pull request merged**